dask/dataframe/methods.py

```diff
@@ -276,12 +276,23 @@ def unique(x, series_name=None):
     return out
 
 
-def value_counts_combine(x):
-    return x.groupby(level=0).sum()
+def value_counts_aggregate(x, sort=True, ascending=False, dropna=None):
```

Comment:
Since this is an internal function, can we define it as `def value_counts_aggregate(x, sort, **groupby_kwargs)`.

Then we can skip the dropna check, since we're the only one calling it and we've already validated things earlier.

Reply:
But then we have to pass or not pass `dropna` in from `value_counts` depending on whether it's None or not. I think having the check in here is clearer.

Reply:
Actually, you are right. That is cleaner.